libs/num/uint.go

```diff
@@ -149,6 +149,10 @@ func (u Uint) Uint64() uint64 {
 	return u.u.Uint64()
 }
 
+func (u Uint) Uint32() uint32 {
```

Comment:
Might be useful to add something like:

```go
func (u Uint) ToInt[T int | int8 | int32 | ...] () T {
     return T(u.Uint64())
}
```
I believe the `constraints` package has a constraint that includes all integers

Reply:
Havent done that. As a matter of fact removed this method because it is not used and needed. must have been left from some previous iteration.